package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "eslint --config .eslintrc.js \"src/\" --ext .js,.jsx,.ts,.tsx && jest --passWithNoTests",
-    "build": "tsc && next build",
```

Comment:
Should `next build` be part of a different script?

Reply:
In https://github.com/inrupt/generate-oidc-token/pull/105, `tsc` is removed altogether with the CLI tool, and entirely replaced by `next build`. In this case, the NexJS app is just a template, so it's not a big deal not to include `next build` I think.